pkg/landscaper/controllers/installations/controller_test.go

```diff
@@ -81,6 +81,30 @@ var _ = Describe("Installation Controller", func() {
 			Expect(inst.Status.InstallationPhase).To(Equal(lsv1alpha1.InstallationPhases.Succeeded))
 		})
 
+		It("should reconcile an installation with reconcile-if-changed annotation", func() {
+			// We consider an Installation in a finished state.
+			// The Installation has no reconcile annotation. Therefore, a reconciliation should have no effect.
+			// After the reconciliation, the jobID and jobIDFinished should be the same as before.
```

Comment:
The comment is wrong, copied from the previous test.

Reply:
ups - I have fixed this